tests/functional/cloudformation/test_deploy.py

```diff
@@ -113,33 +118,15 @@ def test_parameter_overrides_from_inline_cf_like_json(self):
         self._assert_parameters_parsed()
 
     def test_parameter_overrides_from_cf_like_json_file(self):
-        cf_like_json = '''
-            [
-                {"ParameterKey": "Key1", "ParameterValue": "Value1"},
-                {"ParameterKey": "Key2", "ParameterValue": "Value2"}
-            ]
-        '''
-        path = self.files.create_file('param.json', cf_like_json)
+        cf_like_json = [
+            {'ParameterKey': 'Key1', 'ParameterValue': 'Value1'},
+            {'ParameterKey': 'Key2', 'ParameterValue': 'Value2'}
+        ]
+        path = self.create_json_file('param.json', cf_like_json)
         self.command += ' --parameter-overrides file://%s' % path
         self.run_cmd(self.command)
         self._assert_parameters_parsed()
 
-    def test_parameter_overrides_from_invalid_cf_like_json_file(self):
```

Comment:
We should probably keep this test but just make sure we assert that the new error message is coming back.